### ISSUE TITLE
Performance of the model in the wild with Human 3.6M 2d keypoint

### ISSUE THREAD

**Issue by DeepRunner** [facebookresearch/VideoPose3D#17]
Hi. From the repos and experiments,  I find the model performs well in the wild, when 2d keypoints are provided by Mask R-CNN and CPN in the COCO format.  But I wonder
1)whether the model still performs well when 2d keypoints in the wild  are provided in the Human 3.6M format.     
2)Does Mask R-CNN and CPN finetuned on Human3.6 give a good 2d estimation?
Thank you very much.

**Comment by dariopavllo** [comment]
Hi,

For the videos in the wild we sticked with the simplest approach and used pretrained keypoint detectors. Since the generated videos were already very good, we didn't go further. Using the models pretrained on COCO makes sense, since this dataset is very diverse and would thus generalize well.

Anyway, I think it would also make sense to try a fine-tuned model. In theory it should work well, but it's hard to evaluate quantitatively since there is no ground truth.

At least from what we observed, CPN is very easy to fine tune and just a few iterations are required, so I don't think this will affect generalization.
